recipes/aws-c-event-stream/all/conanfile.py

```diff
@@ -39,13 +40,15 @@ def configure(self):
         del self.settings.compiler.libcxx
 
     def requirements(self):
-        self.requires("aws-c-common/0.4.25")
-        self.requires("aws-checksums/0.1.5")
+        self.requires("aws-checksums/0.1.11")
+        if tools.Version(self.version) < "0.2.2":
+            self.requires("aws-c-common/0.6.7")
```

Comment:
`aws-c-common` is still required in 0.2.X (https://github.com/awslabs/aws-c-event-stream/blob/v0.2.7/CMakeLists.txt#L76)

Reply:
Yes but since it is required by aws-c-io I didn't add it. Shall I add it? I didn't do it for my latest PR: https://github.com/conan-io/conan-center-index/blob/master/recipes/aws-c-io/all/conanfile.py#L41-L44

Reply:
`aws-c-common` is a direct dependency of `aws-c-event-stream` (https://github.com/awslabs/aws-c-event-stream/blob/v0.2.7/include/aws/event-stream/event_stream.h#L9) so it must be referenced even if another dependency is already referencing it.